packages/components/src/Form/Fields/FieldRadio/FieldRadio.test.tsx

```diff
@@ -30,10 +30,46 @@ import { renderWithTheme } from '@looker/components-test-utils'
 import { screen } from '@testing-library/react'
 import { FieldRadio } from './FieldRadio'
 
-test('A FieldRadio disabled', () => {
-  renderWithTheme(
+test('FieldRadio renders with label', () => {
+  renderWithTheme(<FieldRadio id="FieldRadioID" label="FM" name="thumbsUp" />)
+
+  expect(screen.getByLabelText('FM')).toBeInTheDocument()
+})
+
+test('FieldRadio renders description and detail props', () => {
+  const { getByText } = renderWithTheme(
+    <FieldRadio
+      description="describe something here."
+      detail="0/50"
+      id="FieldRadioID"
+      label="FM"
+      name="thumbsUp"
+    />
+  )
+  expect(getByText('describe something here.')).toBeInTheDocument()
+  expect(getByText('0/50')).toBeInTheDocument()
+})
+
+test('FieldRadio renders disabled', () => {
+  const { getByRole } = renderWithTheme(
     <FieldRadio disabled id="FieldRadioID" label="FM" name="thumbsUp" />
   )
+  const RadioInput = getByRole('radio')
+
+  expect(RadioInput as HTMLInputElement).toBeDisabled()
```

Comment:
The `as HTMLInputElement` shouldn't be needed (and doesn't appear to be). In general the test can be made more terse like so:

```suggestion
  renderWithTheme(
    <FieldRadio disabled id="FieldRadioID" label="FM" name="thumbsUp" />
  )

  expect(screen.getByRole('radio')).toBeDisabled()
```

Reply:
We've also tried to switch to using `screen.X` instead of destrucuring helpers from RTL's `render` method.